vue-frontend/src/components/android-app-development/DevelopmentSection.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <div class="tw-mt-16 tw-flex tw-flex-col xll:tw-container xll:tw-mt-[4rem]">
+    <div class="tw-container tw-mb-2.5 tw-flex tw-flex-col tw-text-center">
+      <span
+        class="tw-mb-2.5 tw-font-inter-bold tw-text-[1.875rem] tw-leading-[2.4375rem] tw-text-black-core/[0.87] md:!tw-mx-0 md:!tw-w-[100%] md:tw-text-[3.4375rem] md:tw-leading-[5.15625rem] xs:tw-mx-auto xs:tw-w-[71%]"
+      >
+        Android App Development Services</span
+      >
+      <span
+        class="tw-font-inter-regular tw-text-[1rem] tw-leading-[1.5rem] tw-text-black-core/[0.60] md:tw-font-inter-medium md:tw-text-[1.5rem] md:tw-leading-[2.25rem] lg:tw-px-20 2xl:tw-px-56"
+        >With over a decade of expertise in Android app development, our team
+        can help you craft high- quality, user-friendly apps that drive business
+        growth and customer engagement.</span
+      >
+    </div>
+    <div class="swiper-content tw-mt-[1rem]">
+      <swiper
+        :slidesPerView="1.1"
+        :centeredSlides="true"
+        :loop="true"
+        :spaceBetween="10"
+        class="swiper-container"
+        :breakpoints="{
+          '768': {
+            slidesPerView: 1.3,
+          },
+          '1200': {
+            slidesPerView: 1.6,
+          },
+          '1400': {
+            slidesPerView: 1.9,
+          },
+          '1600': {
+            slidesPerView: 2.1,
+          },
+          '1800': {
+            slidesPerView: 2.4,
+          },
+          '2200': {
+            slidesPerView: 2.8,
+          },
+          '2450': {
+            slidesPerView: 1.6,
+          },
+        }"
+      >
+        <swiper-slide
```

Comment:
Last slide be display fully

![Screenshot from 2023-07-10 15-44-43](https://github.com/canopas/canopas-website/assets/69897605/2a80d222-d4e0-47ab-95e2-428316dafc5a)